src/main/java/com/example/auctionapp/Util/MappingUtility.java

```diff
@@ -62,7 +62,9 @@ public static ProductDto mapProductToProductDto(Product product) {
                 images,
                 product.getFeature(),
                 product.getUser().getId(),
-                bids
+                bids,
+                product.getColor().label,
+                product.getSize().label
```

Comment:
It is bad practice anywhere to have direct access to a property, label can be accessed through a getter method